.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: node_js
 node_js:
-  - 0.10
+  - 5.0.0
 before_script:
   - npm install -g bower karma-cli
   - bower install
+  - cd bower_components/modernizr
+  - npm install
+  - ./bin/modernizr -c lib/config-all.json
+  - cd ../..
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+addons:
+  firefox: latest
```

Comment:
Yes, it is. It's necessarily that it doesn't work in old Firefox, it's just something that stops us from passing tests in there. Maybe they don't you allow to set built-in variables on `window` there?